models/solutions.js

```diff
@@ -77,13 +77,12 @@ module.exports = {
           type : Array,
           index : true
         },
-        roles : [{
-          _id : "ObjectId",
-          code : {
```

Comment:
@Prajwal17Tunerlabs please change scope to scope: {
      type: Object,
      default: {},
    },